config/config_test.go

```diff
@@ -599,26 +613,41 @@ func (s *testConfigSuite) TestSecurityValid(c *C) {
 	}
 	for _, tt := range tests {
 		c1.Security.SpilledFileEncryptionMethod = tt.spilledFileEncryptionMethod
-		c.Assert(c1.Valid() == nil, Equals, tt.valid)
+		require.Equal(t, tt.valid, c1.Valid() == nil)
 	}
 }
 
-func (s *testConfigSuite) TestTcpNoDelay(c *C) {
+func TestTcpNoDelay(t *testing.T) {
+	t.Parallel()
+
 	c1 := NewConfig()
 	//check default value
-	c.Assert(c1.Performance.TCPNoDelay, Equals, true)
+	require.True(t, c1.Performance.TCPNoDelay)
 }
 
-func (s *testConfigSuite) TestConfigExample(c *C) {
+func TestConfigExample(t *testing.T) {
 	conf := NewConfig()
 	_, localFile, _, _ := runtime.Caller(0)
 	configFile := filepath.Join(filepath.Dir(localFile), "config.toml.example")
 	metaData, err := toml.DecodeFile(configFile, conf)
-	c.Assert(err, IsNil)
+	require.NoError(t, err)
 	keys := metaData.Keys()
 	for _, key := range keys {
 		for _, s := range key {
-			c.Assert(ContainHiddenConfig(s), IsFalse)
+			require.False(t, ContainHiddenConfig(s))
 		}
 	}
 }
+
+func TestInitializeConfig(t *testing.T) {
```

Comment:
where this new test comes from?

Reply:
I find the func `InitializeConfig` is not tested according to the test coverage, so I add it on my own. I will remove it if unnecessary.

Reply:
No. Nice work.